src/VS.Web.CG.EFCore/ModelMetadata.cs

```diff
@@ -133,7 +133,7 @@ public INavigationMetadata[] Navigations
                 if (_navigations == null)
                 {
                     _navigations = EntityType.GetNavigations()
-                        .Where(n => n.IsDependentToPrincipal() == true && n.ForeignKey.Properties.All(p => !p.IsShadowProperty()))
+                        .Where(n => n.IsOnDependent == true && n.ForeignKey.Properties.All(p => !p.IsShadowProperty()))
```

Comment:
@bricelam - Is this the right replacement function?

Reply:
it is as per https://github.com/dotnet/efcore/blob/master/src/EFCore/Extensions/NavigationExtensions.cs